CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.2.2
+
+* Updated several dependencies [#10][]
```

Comment:
Not the most useful changelog message.

Makes me wonder if the dependencies are too tight, if we have to keep doing that.

In general, I think linking to PRs from changelogs is a waste of time. If the changelog is suitably descriptive, you don't need to go read the PR. And if you do, it's easy enough to find by traversing git history.